### ISSUE TITLE
Collapsed report cards text

### ISSUE THREAD

**Issue by clari182** [responsible-ai-collaborative/aiid#1691]
When collapsed, the text sometimes extends under the report's image. 
Ideally, all report cards should all be similar with their text on the right when collapsed
<img width="912" alt="image" src="https://user-images.githubusercontent.com/4105289/223203937-406f5137-09fa-4ee0-b496-8dff868d8de4.png">

**Comment by lmcnulty** [comment]
I think it's fine if the lines wrap, it's just that if it does, we should try to set the preview text to a length where it's more than one lonely line on its own. I'd also like for the bottom of the image to line up with the bottom of a line of text.

![image](https://user-images.githubusercontent.com/25443411/223218072-a3c210ed-01ba-476c-8980-a8dde5740c17.png)

It looks uncomfortable when the last line is almost far down enough to wrap but not quite:

![image](https://user-images.githubusercontent.com/25443411/223218668-a6d10597-6f38-4c4a-8a9a-430ae1624c3d.png)

**Comment by clari182** [comment]
Yeah, the issue is that these are paragraphs wrapped around the image, which is made with a Markdown so I can't know how many lines displayed there are and if only one or more are going under the image. 
It also needs to stay the same when the card is expanded, without the text moving anywhere. 

I was thinking that maybe when the page loads and all reports are collapsed, the image on the left should have the height of the card and only when expanded the text would go under it.

Otherwise, there are too many edge cases for it to look good on all screen sizes/text structures etc.

But if you have an approach to this feel free to go ahead, I don't want to hold this issue back if I can't figure a solution out!
